cmd/options/controller_options.go

```diff
@@ -33,10 +33,13 @@ type ControllerOptions struct {
 	ExtraVolumeTags map[string]string
 	// ID of the kubernetes cluster.
 	KubernetesClusterID string
+	// AWS Assume Role ARN
+	AWSAssumeRole string
 }
 
 func (s *ControllerOptions) AddFlags(fs *flag.FlagSet) {
 	fs.Var(cliflag.NewMapStringString(&s.ExtraTags), "extra-tags", "Extra tags to attach to each dynamically provisioned resource. It is a comma separated list of key value pairs like '<key1>=<value1>,<key2>=<value2>'")
 	fs.Var(cliflag.NewMapStringString(&s.ExtraVolumeTags), "extra-volume-tags", "DEPRECATED: Please use --extra-tags instead. Extra volume tags to attach to each dynamically provisioned volume. It is a comma separated list of key value pairs like '<key1>=<value1>,<key2>=<value2>'")
 	fs.StringVar(&s.KubernetesClusterID, "k8s-tag-cluster-id", "", "ID of the Kubernetes cluster used for tagging provisioned EBS volumes (optional).")
+	fs.StringVar(&s.AWSAssumeRole, "aws-assume-role", "", "AWS ARN for the controller to assume-role into (optional).")
```

Comment:
I think AWS is redundant here but I guess we can keep it. But we should definitely have "arn" in this option, eg, `-aws-assume-role-arn`. We should also briefly explain what happens the operator doesn't provide this value since it's optional.

Reply:
I put a brief something